incubator/exporter-otlp/src/main/java/io/aklivity/zilla/runtime/exporter/otlp/config/OtlpOverridesConfig.java

```diff
@@ -19,10 +19,13 @@
 public class OtlpOverridesConfig
```

Comment:
Let's remove this (also from schema) and use `OltpConfiguration` for overrides instead.

Reply:
done